apps/teams-test-app/src/components/DialogAPIs.tsx

```diff
@@ -25,6 +36,35 @@ const DialogAPIs = (): ReactElement => {
     setResizeRes('Teams client SDK call dialog.resize was called');
   };
 
+  const sendMessageToChild = (message: string): void => {
+    if (childWindowRef.current && childWindowRef.current !== null) {
+      const childWindow = childWindowRef.current as IAppWindow;
```

Comment:
curious -> why do you need the `as IAppWindow` part? Are you getting an error without it?

Reply:
No, I think its the repercussion of adding type later. I will remove it